### PR TITLE
Add 17-app.md and 18-releases.md

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Guide: Deploy / Releases](https://trello.com/c/0woKGqN1/1105-guide-deploy-releases)

File: _guide/17-app.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: Create and deploy an app
+permalink: /guide/app/
+phase: deploy
+---
+
+# Creating our app
```
Comment (ddollar) [inline _guide/17-app.md]: Maybe "an app"? Not sure how this voice carries through the rest of the guide.

File: _guide/18-releases.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+title: Releases and rollbacks
+permalink: /guide/releases/
+phase: deploy
+---
+
+## Releases
+
+Each time you deploy with `convox deploy`, a new release is created.
```
Comment (ddollar) [inline _guide/18-releases.md]: Or when you update the app's env

File: _guide/18-releases.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+And we can see our app is `updating`:
+
+```
+$ convox apps
+APP           STATUS
+convox-guide  updating
+```
+
+Once the app is `running` again, refresh your web browser to see your change has taken effect:
```
Comment (ddollar) [inline _guide/18-releases.md]: Maybe something about the bits that have to happen to move from updating to running? ECS service has to stabilize, etc

**Comment by ianmalott** [comment]
Whether we do it before merging, or at the end of the week in a cleanup pass, it'd be nice to the various pages of the guide into better alignment.

```
# Early section
Build
  Images
  Services
  Environment

# Later section
Deploy
  Installing a Rack
  Create and deploy an app
  Releases and rollbacks
```

I'd expect the Deploy section to look more like this given the precedent. Could go the other way too, given a new consensus.

```
Deploy
  Rack
  Apps
  Releases
```

Same goes for the structure of the pages. The Apps and Releases pages are missing the initial "What is it?" and "Why does it matter?" explanations included in the earlier pages.

**Comment by soulshake** [comment]
@ianmalott I pasted/replied to your comment on the [Trello card](https://trello.com/c/wR3mY4lQ/954-guide-and-doctor). Merging now, but the stuff pointed out in your comment TBD for sure.

**Pull request merged**